scene/3d/cpu_particles_3d.cpp

```diff
@@ -1409,6 +1409,11 @@ void CPUParticles3D::convert_from_particles(Node *p_particles) {
 	set_emission_shape(EmissionShape(material->get_emission_shape()));
 	set_emission_sphere_radius(material->get_emission_sphere_radius());
 	set_emission_box_extents(material->get_emission_box_extents());
+	set_emission_ring_height(material->get_emission_ring_height());
+	set_emission_ring_radius(material->get_emission_ring_radius());
+	set_emission_ring_inner_radius(material->get_emission_ring_inner_radius());
+	set_emission_ring_cone_angle(material->get_emission_ring_cone_angle());
+	
```

Comment:
There seems to be a trailing tab on this line that should be removed to follow the style guide.

You can make sure to apply style changes automatically by setting up `pre-commit` locally with:
```
pip install pre-commit
pre-commit install
```
in the Godot repo.